src/expertsystem/reaction/particle.py

```diff
@@ -37,42 +35,15 @@
 from particle import Particle as PdgDatabase
 from particle.particle import enums
 
+from .conservation_rules import GellMannNishijimaInput, gellmann_nishijima
+from .quantum_numbers import EdgeQuantumNumbers, Parity, _to_fraction
+
 try:
     from IPython.lib.pretty import PrettyPrinter
 except ImportError:
     PrettyPrinter = Any
 
 
-@total_ordering
-@attr.s(frozen=True, repr=False, eq=False, hash=True)
-class Parity:
```

Comment:
Oh i thought this was already moved in the previous PR...

Reply:
No, the previous PR was only moving the module